src/components/ShareDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  EmailShareButton,
+  EmailIcon,
+  LinkedinShareButton,
+  LinkedinIcon,
+  RedditShareButton,
+  RedditIcon,
+  TwitterShareButton,
+  TwitterIcon,
+} from 'react-share';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    background: theme.palette.background.secondary,
+    color: 'white',
+    fontSize: '15px',
+    textAlign: 'center',
+  },
+  shareIcon: {
+    maringRight: '15px',
+  },
+}));
+
+export default function ShareDialog(props) {
+  const classes = useStyles();
+  const { open, onClose, link, title } = props;
+
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <DialogTitle className={classes.root}>Share</DialogTitle>
+      <DialogContent className={classes.root}>
+        <DialogContentText className={classes.root}>
+          Currently Sharing: {title}
+        </DialogContentText>
+        <EmailShareButton url={link}>
```

Comment:
I recommend adding padding and/or margin between the buttons using `theme.padding(N)` in the style class.

Reply:
Ok yes got that done, initially I intended to get it done but there was an issue with my spelling but its on there now